spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-stream-binder-functional-sample/pom.xml

```diff
@@ -11,7 +11,7 @@
 
   <artifactId>spring-cloud-gcp-pubsub-stream-binder-functional-sample</artifactId>
   <packaging>pom</packaging>
-  <name>Spring Cloud GCP Pub/Sub Stream Binder Functional Code Sample</name>
+  <name>Spring Cloud GCP Code Sample - Pub/Sub Stream Binder</name>
```

Comment:
`Spring Cloud GCP Code Sample - Pub/Sub Stream Binder (Functional style)` because we have another one with legacy annotation style.

Reply:
Done.